packages/multi-select-combo-box/test/basic.test.js

```diff
@@ -115,6 +115,13 @@ describe('basic', () => {
       expect(comboBox.selectedItems).to.deep.equal(['apple']);
     });
 
+    it('should not un-select item when typing its value manually', async () => {
+      await sendKeys({ down: 'ArrowDown' });
```

Comment:
```suggestion
      comboBox.selectedItems = ['orange'];
      await sendKeys({ down: 'ArrowDown' });
```

I think this is missing the initial selection.